product-service/handlers/createProduct.js

```diff
@@ -0,0 +1,63 @@
+import { Client } from 'pg';
+
+const { PG_HOST, PG_PORT, PG_DATABASE, PG_USERNAME, PG_PASSWORD } = process.env;
+const dbOptions = {
+  host: PG_HOST,
+  port: PG_PORT,
+  database: PG_DATABASE,
+  user: PG_USERNAME,
+  password: PG_PASSWORD,
+  ssl: {
+    rejectUnauthorized: false
+  },
+  connectionTimeoutMillis: 5000,
+};
+
+export const createProduct = async (event) => {
+  console.log(event);
+  const client = new Client(dbOptions);
+  await client.connect();
+
+  try {
+    try {
+      const body = JSON.parse(event.body);
+      console.log(body)
+      const { title, description, price, image } = body;
+
+      const { rows: product } = await client.query(`insert into products (title, description, price, image) values ('${title}', '${description}', ${price}, '${image}') returning *`);
+
```

Comment:
From [documentation ](https://node-postgres.com/features/queries)you can used as 
![image](https://user-images.githubusercontent.com/48594817/115692230-74b65f00-a367-11eb-859f-531c33192ecc.png)

Reply:
My code works great

Reply:
Yes, but your code is vulnerable to SQL-injections. Parametrized queries solve that :)